plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtConsoleProxyLoadCommandWrapper.java

```diff
@@ -49,6 +49,11 @@ public Answer executeProxyLoadScan(final Command cmd, final long proxyVmId, fina
             final URL url = new URL(sb.toString());
             final URLConnection conn = url.openConnection();
 
+            // setting TIMEOUTs to avoid possible waiting until death situations
```

Comment:
@wilderrodrigues 

You may also want to get this into the libvirt refactor work you're doing.

This is a copy/paste from:
git/rsafonseca/cloudstack/plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/wrapper/citrix/CitrixConsoleProxyLoadCommandWrapper.java